appraisal.lic

```diff
@@ -95,6 +113,38 @@ class Appraisal
     waitrt?
     bput('wear my zill', 'You slide')
   end
+
+  def appraise_focus(item)
+    case bput("get my #{item}", "^You get.*#{item}", '^What were you referring', 'But that is already', 'You are already')
+    when 'But that is already', '^What were you referring'
```

Comment:
💡 Does `But that is already` mean you already have the item out? If so, could we proceed instead of exiting?

Reply:
This would be an issue with it as an item in your inventory that isn't held, I believe. This is just one of the cases that was listed, I couldn't get it to trigger, or rather didn't even try.